src/robusta/core/playbooks/internal/crds.py

```
@@ -0,0 +1,335 @@
+import logging
+import re
+import subprocess
+from typing import Optional, List
+
+from core.reporting.consts import SlackAnnotations
+from robusta.core.playbooks.common import get_resource_events_table
+from robusta.core.model.env_vars import KUBECTL_CMD_TIMEOUT_SEC
+from robusta.core.model.base_params import ActionParams
+from robusta.core.playbooks.actions_registry import action
+from robusta.core.model.events import ExecutionBaseEvent
+from robusta.core.reporting.blocks import FileBlock, JsonBlock, TableBlock, MarkdownBlock
```

Flake8 (7.3.0)

[error] 'robusta.core.reporting.blocks.TableBlock' imported but unused

(F401)